xblockmufi/private/text_editor.less

```diff
@@ -0,0 +1,221 @@
+@font-face {
+  font-family: "Andron Scriptor Web";
+  src: url('fonts/AndronScriptorWeb.eot');
+  src: url('fonts/AndronScriptorWeb.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('fonts/AndronScriptorWeb.woff') format('woff'), /* Pretty Modern Browsers */
+       url('fonts/AndronScriptorWeb.ttf')  format('truetype'), /* Safari, Android, iOS */
+}
+
+.xblockmufi_block {
+    position: relative;
+    border: 1px solid #ccc;
+    background-color: white;
+    font-size: 18px;
+
+    .toolbar {
+        display: block;
+        position: relative;
+        width: 100%;
+        margin: 0;
+        padding: 0 !important;
+        float: left;
+        border-bottom: 1px solid #ccc;
+        list-style: none;
+
+        li {
+            position: relative;
+            display: block;
+            margin: 0 !important;
+            float: left;
+            border-right: 1px solid #ccc;
+
+            a {
+                display: block;
+                border: none;
+                padding: 5px 8px;
+                background-color: transparent;
+                font-size: 15px;
+                line-height: 18px;
+                color: #444;
+
+                i {
+                    font-size: 18px;
+                    font-style: normal !important;
+                }
+            }
+
+            &.red-text {
+                a {
+                    color: #dd0000;
+                }
+                &:hover a {
+                    background-color: #dd0000 !important;
+                }
+            }
+            &.green-text {
+                a {
+                    color: #008800;
+                }
+                &:hover a {
+                    background-color: #008800 !important;
+                }
+            }
+            &.blue-text {
+                a {
+                    color: #0000dd;
+                }
+                &:hover a {
+                    background-color: #0000dd !important;
+                }
+            }
+
+            &:hover,
+            &.specialButton.active {
+                a {
+                    background-color: #444;
+                    color: #fff;
+                }
+            }
+
+            &.specialButton {
+                float: right;
+                border-right: none;
+                border-left: 1px solid #ccc;
+
+                a {
+                    padding: 6px 8px 3px 8px;
+                }
+                i {
+                    line-height: 15px;
+                    font-size: 24px;
+                    vertical-align: baseline;
+                }
+
+                .sr .toggle-open {
+                    display: none;
+                }
+            }
+        }
+    }
+
+    .specialCharTable {
+        display: none;
+
```

Comment:
I usually use that to show a CSS attribute I'm going to override in JS later, I'll remove the new line and add a comment.